src/oid.rs

```diff
@@ -204,16 +214,38 @@ fn count_generated_is_big_endian() {
     assert_eq!(0x33u8, oid.bytes()[COUNTER_OFFSET + 2]);
 }
 
-#[test]
-fn test_display() {
-    let id = ObjectId::with_string("53e37d08776f724e42000000").unwrap();
+#[cfg(test)]
+mod test {
+    #[test]
+    fn test_display() {
+        let id = super::ObjectId::with_string("53e37d08776f724e42000000").unwrap();
 
-    assert_eq!(format!("{}", id), "53e37d08776f724e42000000")
-}
+        assert_eq!(format!("{}", id), "53e37d08776f724e42000000")
+    }
 
-#[test]
-fn test_debug() {
-    let id = ObjectId::with_string("53e37d08776f724e42000000").unwrap();
+    #[test]
+    fn test_debug() {
+        let id = super::ObjectId::with_string("53e37d08776f724e42000000").unwrap();
 
-    assert_eq!(format!("{:?}", id), "ObjectId(53e37d08776f724e42000000)")
+        assert_eq!(format!("{:?}", id), "ObjectId(53e37d08776f724e42000000)")
+    }
+    use chrono::{offset::TimeZone, Utc};
```

Comment:
Small nit: could you move this import to the top of the test module and separate the test by a single blank line? Something like this:

```rust
#[cfg(test)]
mod test {
    use chrono::{offset::TimeZone, Utc};

    #[test]
    fn test_display() {
       ...
    }

    #[test]
    fn test_debug() {
       ...
    }

    #[test]
    fn test_timestamp() {
       ...
    }
}
```

Reply:
fixed

Reply:
Thanks!